tests/utils/verification/volumeproperties.go

```diff
@@ -90,17 +95,54 @@ func GetVolumePropertiesDockerCli(volName string, hostname string) string {
 	return op
 }
 
+// VerifyAttachedStatus - verify volume is attached and name of the VM attached
+// is consistent on both docker host and ESX
+func VerifyAttachedStatus(name, hostName, esxName string) bool {
+	log.Printf("Confirming attached status for volume [%s]\n", name)
+
+	vmAttachedHost := GetVMAttachedToVolUsingDockerCli(name, hostName)
+	vmAttachedESX := GetVMAttachedToVolUsingAdminCli(name, esxName)
+
+	return (vmAttachedHost == vmAttachedESX)
```

Comment:
as discussed offline .. let's add inline comment that no way to know about expected vmname and need to rely on vmAttachedHost == vmAttachedESX until we come up with govmomi integration to know about the vm name.